packages/ui/package.json

```diff
@@ -18,12 +19,24 @@
     "@repo/typescript-config": "*",
     "@turbo/gen": "^1.12.4",
     "@types/node": "^20.11.24",
-    "@types/react": "18.3.1",
+    "@types/react": "^18.3.12",
     "@types/react-dom": "18.3.1",
     "typescript": "5.5.4"
   },
   "dependencies": {
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1"
+    "@radix-ui/react-accordion": "^1.2.2",
+    "@radix-ui/react-alert-dialog": "^1.1.4",
+    "@radix-ui/react-dialog": "^1.1.4",
+    "@radix-ui/react-popover": "^1.1.4",
+    "@radix-ui/react-slot": "^1.1.1",
+    "@radix-ui/react-switch": "^1.1.2",
+    "@radix-ui/react-tooltip": "^1.1.6",
+    "class-variance-authority": "^0.7.1",
+    "clsx": "^2.1.1",
+    "lucide-react": "^0.471.1",
+    "react": "^18.0.0",
```

Comment:
Is this easy to upgrade to React 19?

Reply:
I was just making ui react version match what is in flame-defi. Do you want to upgrade both of them? The act of upgrading is not hard but we do risk breaking dependencies if we do it.

Reply:
Oh wow, whoops. I upgrade to React 19 but must have done something goofy and not pushed or something. I'll upgrade React